widgets/textInputWidget.js

```diff
@@ -16,13 +16,13 @@ class TextInputWidget extends Widget {
         <div class="widget-title">
           <div class="widget-name">{1}</div>
           <div class="widget-actions">
-            <button class="widget-edit">Edit</button>
-            <button class="widget-delete">Delete</button>
+            <button class="widget-edit btn-icon" title="edit"><i class="far fa-edit"></i></button>
+            <button class="widget-delete btn-icon" title="delete"><i class="fas fa-trash"></i></button>
           </div>
         </div>
         <div class="widget-body">
-          <input id="widget_{0}_text">
-          <button>Send</button>
+          <input class="text-input" id="widget_{0}_text">
```

Comment:
You could wrap this input in a `div class="input-wrapper"` to get the common input styles.
The `width: 100%` and `font-size: 1.25em` are already defined there.